salesagility/SuiteCRM
# Make the pagination buttons on DetailView pages links.

## Description
Rather than buttons with an 'onclick' attribute, links are much better for a number of reasons. Accessibility, ability to hover over the link to see the page it's going to, ability to right click and open in a new tab, etc.

This updates and then updates the CSS to fix the styling of the links.

This is what the pagination buttons look like with this change:

<img width="286" alt="Screen Shot 2019-10-09 at 10 32 55 AM" src="https://user-images.githubusercontent.com/2977353/66501196-2fca2080-ea80-11e9-888c-0b95ba158810.png">

<img width="264" alt="Screen Shot 2019-10-09 at 10 33 01 AM" src="https://user-images.githubusercontent.com/2977353/66501198-2fca2080-ea80-11e9-9c44-431e35ba3056.png">

It should be pretty much the same as before.

One thing I'm not sure about is the `SugarVCR.tpl` file in `include/EditView/`. This PR edits `themes/SuiteP/include/EditView/SugarVCR.tpl`, but I don't know if `include/EditView/SugarVCR.tpl` is used anywhere? Should I also update it? It doesn't seem to have the new icon style either.

## How To Test This
1. Go to the ListView for a module (this is necessary for the pagination to show up)
2. Click a record
3. On the DetailView, see that there are pagination buttons.
4. Click them and see that they work, and are now links rather than a JavaScript onclick event.
5. Make sure the 'disabled' pagination buttons work as-expected.

You should also check the EditView to make sure pagination works for it as well as the DetailView.

You may need to do a Quick Repair and Rebuild to get the new CSS after pulling down the branch.

## Types of changes

I don't really know what to call this, honestly.


### Final checklist
